content/manager_webui/application-overview.md

```diff
@@ -9,8 +9,13 @@ weight: 120
 
 The Dashboard page is the landing page when you log into the Cloudify Web interface. 
 
-By default, the page displays the most commonly used widgets, including the number of deployments, plugins and node instances that are being managed by this Cloudify Manager. In addition, other widgets provide an overview of the current and recent processes related to Cloudify Manager.
+By default, the page displays the most commonly used widgets, including the number of deployments, plugins and node instances that are being managed by this Cloudify Manager. In addition, other widgets provide an overview of the current and recent processes related to Cloudify Manager. The first widget is execution that by default present all the executions on cloudify manager but there a option to choose a specific blueprint or deployment and then the widget will present the execution for the relevant blueprint. The second widget is events/logs that present all the logs on cloudify manager. That widget is similar to the previous one by the option to choose one blueprint or deployment.
```

Comment:
By default, the page displays the most commonly used widgets, including the number of deployments, plugins, and node instances on this Cloudify Manager.
The other default widgets show an overview of the current and recent processes on the Cloudify Manager, including:
* All of the executions on the Cloudify manager.
* All of the logs on the Cloudify manager.
You can configure these widgets to show executions for a specific blueprint or deployment.